lib/tpm-support-lib.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Documentation       Collection of keywords related to UEFI TPM support
+
+Library             ./tpm-support-lib.py
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+
+*** Keywords ***
+# TODO: Most options are split in 2 lines either because name is too long or
```

Comment:
@m-iwanicki what we should do with this TODO?

Reply:
I forgot to delete it, currently keyword returns dictionary containing 3 pages of merged options/values

Reply:
@TomaszAIR fixed in [ca43cb6](https://github.com/Dasharo/open-source-firmware-validation/pull/164/commits/ca43cb6284650227c50aca4e3e4cc2c83598b05c)